packages/mineral-ui-tokens/tokens/generic/cursor.json

```diff
@@ -0,0 +1,9 @@
+{
+  "global": {
+    "type": "cursor",
+    "category": "cursor"
+  },
+  "props": [
+    { "name": "cursor_theme", "value": "pointer" }
```

Comment:
The suffix `_theme` seems a bit strange here.  I don't see this convention elsewhere.  Can it just be `cursor` or `cursor_style`?